network/src/p2p/mod.rs

```diff
@@ -15,7 +15,7 @@
 // along with this program.  If not, see <https://www.gnu.org/licenses/>.
 
 mod connection;
-mod handler;
+pub mod handler;
```

Comment:
IMO, Reexporting `ManagingPeerdb` would be better than making the `handler` module public.